test/files/ri009/pass-ri009-alignment-segment-shape-representation.ifc

```diff
@@ -0,0 +1,824 @@
+ISO-10303-21;
+/*  Testfile: pass-ri009-alignment-segment-shape-representation.ifc */
+/*  IFC-Requirement: ri009; Alignment segment shape representation in IFC */
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME ('pass-ri009-alignment-segment-shape-representation.ifc', '2023-03-14T15:36:48', ('bSI Railway Room - Technical Services'), ('aIFC4Rail'), 'Automatic Pass File Production', 'Python+IfcOpenShell', 'BC-002');
+FILE_SCHEMA(('IFC4X3'));
```

Comment:
I believe that this schema identifier is incorrect (similarly for other test files).